Sources/Turf/GeoJSON.swift

```diff
@@ -0,0 +1,358 @@
+import Foundation
+#if !os(Linux)
+import CoreLocation
+#endif
+
+
+public class BaseFeature: Codable {
+    private enum CodingKeys: String, CodingKey {
+        case properties
+    }
+    
+    public var properties: [String : AnyJSONType]?
+    
+    public required init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        properties = try container.decode([String : AnyJSONType]?.self, forKey: .properties)
+    }
+    
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(properties, forKey: .properties)
+    }
+}
+
+public class Feature: BaseFeature {
+    public var type: GeoJSONType
```

Comment:
One of the caveats we’re introducing is that, if you deserialize and serialize again, all the feature IDs are lost. IDs are quite common in GeoJSON, so losing them would be a surprise. Consider adding an `identifier` property to hold the ID.